app/helpers/bookmarks_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BookmarksHelper
+
+  def bookmarks?
+    if params[:controller] == 'bookmarks'
```

Comment:
the method can be rewritten as:
```ruby
  def bookmarks?
    params[:controller] == 'bookmarks'
  end
```
since the `==` returns either true or false already